tests/unified_sorter_sort_dir_conflict/run.sh

```diff
@@ -60,7 +60,7 @@ function prepare() {
 
     sleep 10
     # starts the first second server instance. It should fail, and bring down the changefeed
-    run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --addr "127.0.0.1:8301" --logsuffix 1 --sort-dir /tmp/cdc_sort_1
+    run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --addr "127.0.0.1:8301" --logsuffix 1
```

Comment:
Can you try using a different logsuffix here?

Reply:
it will fail, because `".*ErrConflictingFileLocks.*"` is preferred, so that set the same logsuffix.

Reply:
Can you override `data-dir`?

Reply:
By using `--data-dir` flag, data-dir can be override.